tests/helpers/pathutils.py

```diff
@@ -18,14 +19,14 @@ def get_test_root(name):
 
 def compare_path(left, right, show=True):
     # If there's a unicode/bytes mismatch, make both unicode.
-    if isinstance(left, ptvsd.compat.unicode):
-        if not isinstance(right, ptvsd.compat.unicode):
+    if isinstance(left, unicode):
+        if not isinstance(right, unicode):
             right = right.decode(sys.getfilesystemencoding())
-    elif isinstance(right, ptvsd.compat.unicode):
+    elif isinstance(right, unicode):
         right = right.decode(sys.getfilesystemencoding())
```

Comment:
I think this should be `encode`, since it's meant to go from `unicode` to `bytes` in this branch.